custom_components/solarman/inverter_definitions/zcs_azzurro-hyd-zss-hp.yaml

```diff
@@ -14,6 +14,22 @@ parameters:
   - group: PV
     update_interval: 5
     items:
+      - name: "PV Power"
+        description: "Combined power of all inputs"
+        realtime:
+        class: "power"
+        state_class: "measurement"
+        uom: "W"
+        rule: 1
+        digits: 0
+        registers: [0x0586, 0x0589]
+        sensors:
+          - scale: 10
+            registers: [0x0586]
+          - scale: 10
```

Comment:
Hi @Edsol, you can "move" `scale: 10` up, outside of sensors list. 😉

Reply:
fixed 👍